view/launch.py

```diff
@@ -0,0 +1,5 @@
+import gui
+
+if __name__ == "__main__":
+    GUI = gui() 
+    GUI.to_streamlit()
```

Comment:
I think it should be` GUI.generate_and_show_gui()`. 
`.to_streamlit()` was to add the map iirc.

Reply:
Done